src/ConstructiveSolidGeometry/LinePrimitives/Arc.jl

```diff
@@ -0,0 +1,58 @@
+struct Arc{T,TP,TH} <: AbstractLinePrimitive{T}
+    r::T
+    center::TP
+    α::TH
```

Comment:
Why not call this `φ`?
Then you can also use `get_φ_limits`

Reply:
The purpose of this was to not confuse it with coordinate `φ` since the plane that the arc can be is technically arbitrary